angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -141,8 +141,8 @@ import {
       <nav
         *ngIf="isNarrowNavOpen"
         class="sprk-c-Masthead__narrow-nav"
-        data-sprk-mobile-nav="mobileNav"
         role="navigation"
+        [id]="narrowNavAriaControls"
```

Comment:
I wonder about this because if the user wants to add their own id, aria controls would break

Reply:
This is handled internally, so they only way they could add their own `id` in that specific spot would be to grab it out of the DOM after it's been rendered and update it programatically.